### PR TITLE
Update hook-passlib.py

### PR DESCRIPTION
**Pull request by mhils**
Thank you @Legorooj and other for setting this repo up here! 😃 We've been maintaining some custom third-party hooks for @mitmproxy for a while, which I'd be happy to contribute here.

To start things off, passlib is missing a hidden import. Without `configparser`, importing passlib yields the following traceback:
```
    import passlib.apache
  File "lib/python3.7/site-packages/PyInstaller/loader/pyimod03_importers.py", line 493, in exec_module
    exec(bytecode, module.__dict__)
  File "passlib/apache.py", line 14, in <module>
  File "lib/python3.7/site-packages/PyInstaller/loader/pyimod03_importers.py", line 493, in exec_module
    exec(bytecode, module.__dict__)
  File "passlib/context.py", line 21, in <module>
  File "passlib/utils/compat/__init__.py", line 417, in __getattr__
  File "passlib/utils/compat/__init__.py", line 379, in _import_object
ModuleNotFoundError: No module named 'configparser'
```

### CODE REVIEW

**Comment by bwoodsend** [comment]
> We've been maintaining some custom third-party hooks for @mitmproxy for a while, which I'd be happy to contribute here.

Yes please :smile:.

**Review by bwoodsend — CHANGES_REQUESTED** [review]
This needs a changelog entry. See the [news README](https://github.com/pyinstaller/pyinstaller-hooks-contrib/blob/master/news/README.txt).

Optionally, it would be good if we could have a test for this too...

**Comment by mhils** [comment]
Would you like me to open separate PRs for each hook or update everything here? There will definitely be some merge conflicts in the former case due to tests and requirements being added at the same location.

**Comment by bwoodsend** [comment]
Definitely separate PRs please.

File: news/38.update.rst
Comment (bwoodsend) [inline news/38.update.rst:1]: This is PR #39 so should be called `39.update.rst`. (Not entirely sure what difference it makes.)

**Review by bwoodsend — COMMENTED** [review]
I'll let CI finish, then we're probably good to go.

**Comment by bwoodsend** [comment]
Well tests pass so looks good. Normally we have @Legorooj to give it a final check but he's away. @amifunny Could you give this a once over then merge it if you think it's OK?

**Review by bwoodsend — APPROVED** [review]

**Review by amifunny — APPROVED** [review]
Everything is ok. @bwoodsend Should merge now.

**Comment by bwoodsend** [comment]
@amifunny Go for it.

**Comment by amifunny** [comment]
> @amifunny Go for it.

@bwoodsend You want me to merge? I don't have commit rights.

File: news/38.update.rst
Comment (Legorooj) [reply news/38.update.rst]: The difference is the rendering in the changelog. It links to the number in the filename.

**Comment by Legorooj** [comment]
@bwoodsend I haven't given @amifunny write permission here yet. However, I suggest you get him to review PRs first, before you do, so he can practice.

**Pull request merged**
